flutter-intellij-community.iml

```diff
@@ -12,10 +12,6 @@
       <sourceFolder url="file://$MODULE_DIR$/third_party/intellij-plugins-dart/testSrc" isTestSource="true" />
       <excludeFolder url="file://$MODULE_DIR$/.pub" />
       <excludeFolder url="file://$MODULE_DIR$/build" />
-      <excludeFolder url="file://$MODULE_DIR$/packages" />
```

Comment:
I think this churn goes away with the 2017.2 version of the Dart plugin. Shouldn't matter either way.